spark_stream.py

```diff
@@ -69,7 +69,7 @@ def create_spark_connection():
     try:
         s_conn = SparkSession.builder \
             .appName('SparkDataStreaming') \
-            .config('spark.jars.packages', "com.datastax.spark:spark-cassandra-connector_2.13:3.4.1,"
+            .config('spark.jars.packages', "com.datastax.spark:spark-cassandra-connector_2.13:3.4.1",
```

Comment:
@aviravipati I am not sure what you're trying to do here but it's wrong. When you're adding multiple jars with spark, you have something like `jar1, jar2, jar3` not `jar1 jar2 jar3`. I think the jars works fine as demonstrated in the video, unless there's something else you're trying to achieve that's unclear to me.